dashboard/app/controllers/levels_controller.rb

```diff
@@ -42,6 +42,20 @@ def show
   def edit
   end
 
+  # GET all the information for the mini rubric
+  def get_rubric
```

Comment:
it looks like you've added all the correct authorization actions (on lines 9-12), but just to double-check -- have you tested that a non-levelbuilder teacher/student account can access this endpoint?

Reply:
yup!